src/Validators/PostcodeValidator.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Validators;
+
+class PostcodeValidator extends StringValidator
+{
+    /**
+     * Validates postcode
+     * @param string $postcode
+     * @return string
+     * @throws \Exception
+     */
+    public static function validatePostcode(string $postcode)
+    {
+        $postcode = strtoupper(str_replace(' ','', $postcode));
```

Comment:
Yeah add a validator in string validator to validate all uppercase and then return an exception if not all uppercase,  idont think we should manipulate user data like this on the back-end